users/test.py

```diff
@@ -67,5 +67,101 @@ def test_unkown_user_not_authorized(self):
             MALFORMATTED_IN_DB_USER, self.mock_collection))
 
 
+USER_ID = "valid-user-id"
+USER_NAME = "Valid User Name"
+ADDITIONAL_INFORMATION = "I'm not found in other documents"
```

Comment:
Put all the constants together near the top of the file.

Reply:
Done https://github.com/knative-portability/large-events/pull/8/commits/69a2afea847898f941de63007c5a6a2db7ac0c0b